examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/ConnectionExampleFragment.java

```diff
@@ -152,9 +228,124 @@ public void handle(MatterError err) {
+              Log.i(
+                  TAG,
+                  "displayPasscodeInputDialog() User entered CastingPlayer/Commissioner-Generated passcode: "
+                      + passcodeLongValue);
+            } catch (NumberFormatException nfe) {
+              Log.e(
+                  TAG,
+                  "displayPasscodeInputDialog()User entered CastingPlayer/Commissioner-Generated passcode is not a valid integer. NumberFormatException: "
+                      + nfe);
+              connectionFragmentStatusTextView.setText(
+                  "User entered CastingPlayer/Commissioner-Generated passcode is not a valid integer: "
+                      + passcode
+                      + "\n\n");
+            }
+
+            // Update the CommissionableData DataProvider and AndroidChipPlatform with the user
+            // entered CastingPlayer/Commissioner-Generated setup passcode. This is mandatory for
+            // Commissioner-Generated passcode commissioning since the commissioning session's PAKE
+            // verifier needs to be updated with the entered passcode.
+            InitializationExample.commissionableDataProvider.updateCommissionableDataSetupPasscode(
+                passcodeLongValue, DEFAULT_DISCRIMINATOR_FOR_CGP_FLOW);
+
+            Log.i(TAG, "displayPasscodeInputDialog() calling continueConnecting()");
+            connectionFragmentStatusTextView =
+                getView().findViewById(R.id.connectionFragmentStatusText);
+            connectionFragmentStatusTextView.setText(
+                "Continuing to connect with Casting Player with device name: "
+                    + targetCastingPlayer.getDeviceName()
+                    + "\nCastingPlayer/Commissioner-Generated Setup Passcode: "
+                    + InitializationExample.commissionableDataProvider.get().getSetupPasscode()
+                    + "\nDiscriminator: "
+                    + InitializationExample.commissionableDataProvider.get().getDiscriminator());
+
+            MatterError err = targetCastingPlayer.continueConnecting();
+
+            if (err.hasError()) {
+              MatterError finalErr = err;
+              getActivity()
+                  .runOnUiThread(
+                      () -> {
+                        connectionFragmentStatusTextView.setText(
+                            "Casting Player connection failed due to: " + finalErr + "\n\n");
+                      });
+              Log.e(
+                  TAG,
+                  "displayPasscodeInputDialog() continueConnecting() failed, calling stopConnecting() due to: "
+                      + err);
+              // Since continueConnecting() failed, Attempt to cancel the connection attempt with
+              // the CastingPlayer/Commissioner.
+              err = targetCastingPlayer.stopConnecting();
+              if (err.hasError()) {
+                Log.e(TAG, "displayPasscodeInputDialog() stopConnecting() failed due to: " + err);
```

Comment:
Can we call connectionFragmentStatusTextView.setText() here and set an error status?